cluster/images/etcd/migrate-if-needed.sh

```diff
@@ -132,8 +134,11 @@ ATTACHLEASE="${ATTACHLEASE:-/usr/local/bin/attachlease}"
 ROLLBACK="${ROLLBACK:-/usr/local/bin/rollback}"
 
 # Do the roll-forward migration if needed.
-for step in "${SUPPORTED_VERSIONS[@]}"; do
-  if [ "${step}" == "${CURRENT_VERSION}"  -a "${CURRENT_VERSION}" != "${TARGET_VERSION}" ]; then
+SKIP_STEP=true
```

Comment:
Can you add a comment explaining the logic here?  I think it goes like this:  by default, we skip "start then stop etcd" for all versions.  We don't start doing the "start then stop etcd" section until we're one step *past* the current version.